packages/@lwc/integration-karma/test/mixed-shadow-mode/LightningElement.shadowSupportMode/index.spec.js

```diff
@@ -40,3 +41,26 @@ describe('ENABLE_MIXED_SHADOW_MODE', () => {
         setFeatureFlagForTest('ENABLE_MIXED_SHADOW_MODE', false);
     });
 });
+
+describe('ENABLE_NATIVE_SHADOW_MODE', () => {
+    beforeEach(() => {
+        setFeatureFlagForTest('ENABLE_MIXED_SHADOW_MODE', true);
+    });
+
+    it('should be configured as "any" (sanity)', () => {
+        expect(NativeOnly.shadowSupportMode === 'native').toBeTrue();
+    });
+
+    it('should enable mixed shadow mode', () => {
+        const elm = createElement('x-native-only', { is: NativeOnly });
+        if (process.env.NATIVE_SHADOW_ROOT_DEFINED) {
+            expect(isNativeShadowRootInstance(elm.shadowRoot)).toBeTrue();
+        } else {
+            expect(isSyntheticShadowRootInstance(elm.shadowRoot)).toBeTrue();
```

Comment:
I guess the expectation here is that an exception is thrown? I'm not even sure we test with browsers that don't implement shadow roots natively.